src/controllers/controller.bar.js

```diff
@@ -86,6 +86,10 @@ defaults._set('horizontalBar', {
 	}
 });
 
+function getValue(scale, dataPoint) {
+	return typeof dataPoint === 'string' ? Number(dataPoint) : scale.getRightValue(dataPoint);
```

Comment:
I agree with @etimberg, would be better in `getRightValue()` but only for the `linearbase` impl. (as we did with the `time` scale). I don't think it's silly because `+'1.2'`, `Number('1.2')` or `parseFloat('1.2')` return valid numbers. Linear scales expect numbers from the data, so as long as the values represent valid numbers, I'm not sure why we would want to make it less flexible (we don't know from where the user gets the data, could be from an API that returns numbers as string).